docs/nd4j/templates/quickstart.md

```diff
@@ -0,0 +1,381 @@
+---
+title: Quickstart
+short_title: Quick start tutorial
+description: ND4J Key features and brief samples.
+category: ND4J
+weight: 1
+---
+<!--- Comments are standard html. Tripple dash based on stackoverflow: https://stackoverflow.com/questions/4823468/comments-in-markdown -->
+
+<!--- Borrowing the layout of the Numpy quickstart to get started. -->
+
+## Introduction
+<!--- What is ND4J and why is it important. From the nd4j repo readme.  -->
+ND4J is a scientific computing library for the JVM. It is meant to be used in production environments rather than as a research tool, which means routines are designed to run fast with minimum RAM requirements. The main features are:
+* A versatile n-dimensional array object.
+* Linear algebra and signal processing functions.
+* Multiplatform functionality including GPUs.
+
+This quickstart follows the same layout and approach of the [Numpy quickstart](https://docs.scipy.org/doc/numpy/user/quickstart.html). This should help people familiar with Python and Numpy get started quickly with Nd4J.
+
+## Prerequisites
+<!--- // Java, Maven, git. Coding skills and hello world example. -->
+
+To follow the examples in this quick start you will need to know some Java. You will also need to install the following software on your computer:
```

Comment:
"need to know some Java".
Let's also briefly mention that ND4J can be used in any JVM language (scala, kotlin, etc)